pep/app/pods/components/page/sidebar/widgets/component.ts

```diff
@@ -0,0 +1,37 @@
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+import { WIDGET } from 'pep/constants/sidebar';
+import { action } from '@ember/object';
+interface PageSidebarWidgetsArgs {}
+
+export interface PageSidebarWidgetArgs {
+    toggleIsOpen: (widget: WIDGET) => void;
+    openWidgets: WIDGET[];
+}
+
+export default class PageSidebarWidgets extends Component<PageSidebarWidgetsArgs> {
+    @tracked openWidgets: WIDGET[] = [WIDGET.WHATS_NEW, WIDGET.MOST_CITED, WIDGET.MOST_VIEWED];
+
+    get showCloseAll() {
+        return this.openWidgets.length > 2;
```

Comment:
from a user perspective this felt a bit weird/unexpected for that toggle to change on `> 2` open. I'm not sure which way the logic should go but i think it feels more natural if its on `> 0` open or closed, e.g. either show "Show all" if at least one widget is closed, otherwise show "Close all". Or, show "Close all" if at least one widget is open, otherwise show "Show all".

Reply:
I think it would make the most sense at greater than or equal to 2. If you have one opened, you can just as easily close it the normal way. if you have two opened, we should show close all.

Reply:
yeah i think changing this to `>= 2` is fine too.